elm-package.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
I think I'd like to hold onto the 0.18 and older manifest for a while longer, at least for a couple of version releases. I'd hate to have to force people to upgrade to 0.19 in order to get a bug fix or a feature.

Reply:
Sure, makes sense. I can amend the commit.